cmd/rm-main.go

```diff
@@ -141,7 +141,7 @@ func removeSingle(url string, isIncomplete bool, isFake bool, older int) error {
 		errorIf(pErr.Trace(url), "Invalid argument `"+url+"`.")
 		return exitStatus(globalErrorExitStatus) // End of journey.
 	}
-	isFetchMeta := false
+	isFetchMeta := true
```

Comment:
Why do we need this isFetchMeta variable? It looks like it is being used only in the next line.

Reply:
the explicit flag definition is self documenting. Otherwise we would still need a comment to make the args passed to Stat function clear.